tests/e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -14,30 +14,41 @@ import { DriverHelper } from '../DriverHelper';
 import { CLASSES } from '../../inversify.types';
 import 'reflect-metadata';
 import * as rm from 'typed-rest-client/RestClient';
-import { error } from 'selenium-webdriver';
 import { WorkspaceStatus } from './WorkspaceStatus';
 import { ITestWorkspaceUtil } from './ITestWorkspaceUtil';
+import axios from 'axios';
```

Comment:
This is not a flow of this PR but it would be great to get rid of the "RestClient" library and switch all methods to the "axios".

Reply:
I've removed the RestClient as a part of this PR.